src/Lumper.Lib/BSP/JsonContractResolver.cs

```diff
@@ -1,41 +1,45 @@
 namespace Lumper.Lib.BSP;
+
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Lumper.Lib.BSP.IO;
-using Lumper.Lib.BSP.Lumps;
+using IO;
+using Lumps;
```

Comment:
I went with omitting prefixes since we end up with a *lot* of `Lumper.Lib.IO` etc... I'm pretty agnostic to what approach we take, so long as it's consistent. So if we want to prefix IO with stuff I think we should prefix everything, otherwise it just gets messy and I hate having to fuss with imports and stuff in review. So if you want long version lmk, though I'd prefer shortening. Note that `IO` isn't an existing namespace, the MS one is `System.IO`.

Reply:
I'm used to the long version and I think the autoformater would keep them together nicely.

Reply:
Oh huh you're right, CSharpier orders them automatically. Definitely better to have the whole path then. Probably won't get around to doing that for a couple days but will do eventually

Reply:
Going to do this in the autoformatting PR after this PR is merged.